packages/muelu/src/Transfers/Smoothed-Aggregation/MueLu_TentativePFactory_def.hpp

```diff
@@ -119,13 +119,24 @@ namespace MueLu {
     RCP<MultiVector>      fineNullspace = Get< RCP<MultiVector> >     (fineLevel, "Nullspace");
     RCP<const Map>        coarseMap     = Get< RCP<const Map> >       (fineLevel, "CoarseMap");
 
+    // Sanity checking
+    if(A->getRowMap()->getNodeNumElements() != fineNullspace->getMap()->getNodeNumElements())
+      printf("[%d] A->getRowMap()->getNodeNumElements() = %d fineNullspace->getMap()->getNodeNumElements() %d\n",fineNullspace->getMap()->getComm()->getRank(),
+	     A->getRowMap()->getNodeNumElements(),fineNullspace->getMap()->getNodeNumElements());
+
```

Comment:
Why printf? Shouldn't we use our logging framework?

Reply:
@csiefer2 this could probably be removed, it looks like it was used for debug purpose?
Otherwise we might actually want to throw on that type of errors...